### PR TITLE
fix: review validation to can send flag inside Send View

### PR DESCRIPTION
**Pull request by bradleySuira**
# :books: Linear Ticket :books: #ext-222

### CODE REVIEW

File: src/views/Send/SendNFT.vue [hunk 1]
```diff
@@ -613,7 +615,16 @@ export default {
       this.sendFees = sendFees
     },
     updateSendFees: _.debounce(async function (amount) {
-      await this._updateSendFees(amount)
+      // In NFT screen the balance is equal to the native asset balance
+      console.log(
+        `on updateSendFees => amount: ${amount}, balance: ${this.balance}, asset: ${this.asset}, `
+      )
```
Comment (mpwanyi256) [inline src/views/Send/SendNFT.vue:621]: @bradleySuira I think we forgot to remove this console log

File: src/views/Swap/Swap.vue [hunk 2]
```diff
@@ -211,7 +211,7 @@
               class="btn btn-primary btn-lg"
               id="swap_review_button"
               @click="review"
-              :disabled="!canSwap || cannotCoverNetworkFee"
+              :disabled="false"
```
Comment (mpwanyi256) [inline src/views/Swap/Swap.vue:214]: @bradleySuira Do we need the prop if we are defaulting it to false?

Comment (bradleySuira) [reply src/views/Swap/Swap.vue]: fixed man, thanks!

File: src/views/Send/SendNFT.vue (hunk 1 above)
Comment (bradleySuira) [reply src/views/Send/SendNFT.vue:621]: we will keep this for debugging until we test this a lot

**Pull request merged**